apps/console/src/features/core/configs/routes.tsx

```diff
@@ -393,7 +427,7 @@ export const getAppViewRoutes = (useExtendedRoutes: boolean = false): RouteInter
                             icon: import("../../../extensions/assets/images/icons/paint-palette-and-brush-outline.svg")
                         },
                         id: "branding",
-                        name: "extensions:develop.sidePanel.branding",
+                        name: "Styles & Text",
```

Comment:
Let's add i18n placeholders for the content.

Reply:
fixed https://github.com/wso2/identity-apps/pull/4202/commits/8ab65827fa45aca316b7682aeeb2af8236bfaaa8